doc/sphinx/source/graph-basics.rst

```diff
@@ -1,25 +1,92 @@
 .. graph-basics:
 
+#############
 Graph Basics
-============
+#############
 
-This section describes the basic concepts you need to know when 
-constructing a graph.
 
+This section explains some of the primary concepts used in the nGraph Library.
+Some new ideas regarding our unique departure from the first generation of deep 
+learning software design are also introduced. 
 
-Framework Bridges
-------------------
+Most frameworks for the first-generation of DNNs had a baked-in assumption that 
```

Comment:
This isn't quite right; first-generation DNNs came long before the frameworks. Maybe "Many frameworks have a baked-in assumption..."